include/odp/api/spec/classification.h

```diff
@@ -204,6 +212,12 @@ typedef struct odp_cls_capability_t {
 
 	/** Supported threshold type for BP */
 	odp_threshold_types_t threshold_bp;
+
+	/** Maximum number of mark value.
+	 * ODP_CLS_MARK_INVALID means that marking is not supported.
+	 * @see odp_pmr_create_opt_t::mark
+	 */
+	uint64_t max_mark;
```

Comment:
If maximum is 0, then packet mark value remain always in it's default value (==0). So, marking is kind of always  supported, when max is 0 it's the only supported value.

Reply:
Makes sense. I will reword the "supported" aspect in the next version.